aquacomputer_d5next.c

```diff
@@ -447,13 +501,24 @@ static int aqc_write(struct device *dev, enum hwmon_sensor_types type, u32 attr,
 	switch (type) {
 	case hwmon_pwm:
 		switch (attr) {
+		case hwmon_pwm_enable:
+			if (priv->fan_ctrl_offsets != NULL) {
+				if (val < 0)
+					return val;
```

Comment:
returning the "incorrect" input value isn't semantically correct I think. I'd need to check, but I think `-EINVAL` would be good.

Reply:
fixed in 1dede22